tests/unit/ctest.h

```diff
@@ -79,14 +79,24 @@ struct ctest {
    unsigned int magic;
 };
 
+/*
+ * Message verbosity levels:  Modeled on syslog's severity levels.
+ * See: https://en.wikipedia.org/wiki/Syslog
+ */
+typedef enum msg_level {
+   MSG_LEVEL_SILENT = 0, // No messages are printed / sent to client
+   MSG_LEVEL_ERRORS = 3, // Only errors are printed / sent to client
+   MSG_LEVEL_DEBUG  = 7, // All messages, including debug outputs, are printed
+} msg_level;
+
 // If FALSE (default), then CTEST_LOG_INFO() and CTEST_LOG() will no-op.
-extern _Bool Ctest_verbose;
+extern msg_level Ctest_verbosity;
 
-// For immedate logging to stdout
+// For immediate logging to stdout at any message verbosity level.
 // (contrast with CTEST_LOG which does buffered/delayed logging to stderr)
 #define CTEST_LOG_INFO(...)                                                    \
    do {                                                                        \
-      if (Ctest_verbose) {                                                     \
+      if (Ctest_verbosity != MSG_LEVEL_SILENT) {                               \
```

Comment:
Old code is "fine" but the rewrite is more true to form.

Reply:
And in that spirit, I think `>=` would better convey the overall semantics.

Reply:
But then `>=` implicitly assumes that `MSG_LEVEL_SILENT` is  == 0 . 

W/o relying on the value of this token, it's better to say `!=`;  I think.